src/odemis/driver/test/actuator_test.py

```diff
@@ -1274,6 +1275,225 @@ def tearDown(self):
         self.dev_cycle.terminate()
         super(TestRotationActuator, self).tearDown()
 
+CONFIG_5DOF = {"name": "5DOF",
+               "role": "stage",
+               "ref_on_init": True,
+               "linear_speed": 0.001,  # m/s
+               "locator": "network:sn:MCS2-00001602",
+               "axes": {
+                   'x': {
+                       'range': [-3e-3, 3e-3],
+                       'unit': 'm',
+                   },
+                   'y': {
+                       'range': [-3e-3, 3e-3],
+                       'unit': 'm',
+                   },
+                   'z': {
+                       'range': [-3e-3, 3e-3],
+                       'unit': 'm',
+                   },
+                   'rx': {
+                       'range': [-0.785, 0.785],
+                       'unit': 'rad',
+                   },
+                   'rz': {
+                       'range': [-0.785, 0.785],
+                       'unit': 'rad',
+                   },
+               },
+               }
+TEST_NOHW = True
+if TEST_NOHW:
+    CONFIG_5DOF['locator'] = 'fake'
```

Comment:
I think that here, for testing the *wrapper*, it's fine to just always have it use the simulator. So you can drop this part.

Reply:
I will remove it after finishing testing with the real hardware..